test/helpers.bash

```diff
@@ -499,3 +499,20 @@ function cleanup_network_conf() {
 function temp_sandbox_conf() {
 	sed -e s/\"namespace\":.*/\"namespace\":\ \"$1\",/g "$TESTDATA"/sandbox_config.json > $TESTDIR/sandbox_config_$1.json
 }
+
+function wait_until_exit() {
+	ctr_id=$1
+	# Wait for container to exit
+	attempt=0
+	while [ $attempt -le 10 ]; do
```

Comment:
In the hunks you removed this was `-le 100`.  You probably want to keep that value.  Part of me wants to make it configurable, but we should never be hitting the limit, it's just a cap to get hung tests to fail faster (and Travis would time them out eventually anyway).

Reply:
agree, restored to 100, a test actually required more than 10s